product_by_supplier/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2010-2013 Elico Corp. <lin.yu@elico-corp.com>
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Product by Supplier info',
+    'version': '11.0.1.0.0',
+    'development_status': 'Mature',
+    'category': 'Purchase Management',
+    'summary': 'Show products grouped by suppliers',
+    'author': "Elico Corp,"
+              "Odoo Community Association (OCA)",
+    'website': 'http://www.elico-corp.com, ',
```

Comment:
@lreficent Change website URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)

Reply:
Done